heat/templates/multiServerIPv6withLBaaSv2_octavia/README.md

```diff
@@ -0,0 +1,101 @@
+# Two servers with IPv4 and IPv6 behind a Octavia-LoadBalancer
+
+The following resources will be created by the template:
+
+* A new network
+* Two subnets within the network
+  * The first subnet is an IPv4 subnet with a hardcoded cidr: 10.100.0.0/16
+  * The second subnet is an IPv6 subnet, dynamically generated from an existing subnet pool
+  * The name or ID of the subnet pool can be updated in the ubuntu_env.yml file
+* A router with the new subnets assigned and internet connection
+* A new security group with the following rules:
+  * Ingress ICMP for IPv4 and IPv6 from any destination
+  * Ingress SSH for IPv4 and IPv6 from any destination
+* IPv4 LoadBalancer
+* IPv6 LoadBalancer
+* FloatingIP (IPv4) associated to the IPv4 LoadBalancer
+* A ResourceGroup with ...
+  * `count` servers with two interfaces each (1. IPv4, 2. IPv6, to get the ordering for LoadBalancer Members right)
+  * Each server runs [emilevauge/whoamI](https://github.com/emilevauge/whoamI) listening on port `8000`
+  * LoadBalancer Member for each server on IPv4 LoadBalancer
+  * LoadBalancer Member for each server on IPv6 LoadBalancer
+
+## Requirements
+
+* Ubuntu 18.04 (it won't work with Ubuntu 12.04 or 14.04)
```

Comment:
I would guess, Ubuntu 16.04 doesn't work as well, because of the netplan config.